client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/eligibility-warnings-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Dialog } from '@automattic/components';
+import EligibilityWarnings from 'calypso/blocks/eligibility-warnings';
+import type { SiteDetails } from '@automattic/data-stores';
+import './eligibility-warnings-modal.scss';
+
+export const EligibilityWarningsModal = ( {
+	site,
+	isMarketplace,
+	isOpen,
+	handleClose,
+	handleContinue,
+}: {
+	site?: SiteDetails;
+	isMarketplace?: boolean;
+	isOpen: boolean;
+	handleClose: () => void;
+	handleContinue: () => void;
+} ) => {
+	return (
+		<Dialog
+			additionalClassNames="eligibility-warnings-modal__dialog-content"
+			isVisible={ isOpen }
+			onClose={ handleClose }
+			showCloseIcon={ true }
+		>
+			<EligibilityWarnings
+				siteId={ site?.ID }
+				currentContext="plugin-details"
```

Comment:
Non-blocking: I'm unsure how we use this, but the `currentContext` should probably be different here.

Reply:
That `currentContext` as well as `backUrl` are used in [this logic](https://github.com/Automattic/wp-calypso/blob/07b49d2ed47b724e92eba97db96730d7046edc14/client/blocks/eligibility-warnings/index.tsx#L423) to, [later on](https://github.com/Automattic/wp-calypso/blob/07b49d2ed47b724e92eba97db96730d7046edc14/client/blocks/eligibility-warnings/warning-list.tsx#L93), define the title and the wording used. I agree is confusing and removing them also displays a wording that is correct for this scenario 
<img width="699" alt="CleanShot 2023-09-15 at 16 20 35@2x" src="https://github.com/Automattic/wp-calypso/assets/3519124/3d391438-b09c-4121-91a5-fe34ab86feef">
so I have gone ahead and made `backUrl` as optional and removed passing `currentContext` and `backUrl` to the modal in this commit https://github.com/Automattic/wp-calypso/pull/81412/commits/07b49d2ed47b724e92eba97db96730d7046edc14. Please let me know your view on this.

Reply:
Great, it looks cleaner now.